src/lib.rs

```diff
@@ -51,18 +53,22 @@ impl Tera {
         let mut templates = BTreeMap::new();
 
         // We are parsing all the templates on instantiation
-        for entry in WalkDir::new(dir).into_iter().filter_map(|e| e.ok()) {
-            let path = entry.path();
+        for entry in glob(dir).unwrap().filter_map(|e| e.ok()) {
+            let path = entry.as_path();
             // We only care about actual files
             if path.is_file() {
+                debug!("Found file: {:?}", path);
```

Comment:
you marked env_logger as dev dependencies but this would be run in release as well and doesn't the logger need to be initialised?